File: Java Basics/IntroductionToJava/src/Exams/HalfSum.java
package Exams;

import java.util.Scanner;

public class HalfSum {

	public static void main(String[] args) {
		Scanner input = new Scanner(System.in);
		int n = input.nextInt();
		int[] numbers = new int[2 * n];
		
		for (int i = 0; i < 2 * n; i++) {
			numbers[i] = input.nextInt();
		}
		
		int firstSum = 0;
		int secondSum = 0;
		
		for (int i = 0; i < numbers.length / 2; i++) {
			firstSum += numbers[i];
		}
		for (int i = n / 2; i < numbers.length; i++) {
			secondSum += numbers[i];
		}
		
		if (firstSum == secondSum) {
			System.out.println("Yes, sum = " + firstSum);
		}
		else if (firstSum > secondSum) {
			System.out.println("No, diff = " + (firstSum - secondSum));
		}
		else if (firstSum < secondSum) {
			System.out.println("No, diff = " + (secondSum - firstSum));
		}
	}

}

File: C# Basics/SoftUni Exams/02.Half Sum/HalfSum.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace _02.Half_Sum
{
    class HalfSum
    {
        static void Main(string[] args)
        {
            int n = int.Parse(Console.ReadLine());
            int doubleN = n * 2;
            int counter = 0;
            int firstSum = 0;
            int secondSum = 0;
            for (int i = 0; i < doubleN; i++)
            {
                int j = int.Parse(Console.ReadLine());
                if (counter < n)
                {
                    firstSum += j;
                }
                else
                {
                    secondSum += j;
                }
                counter++;
            }
            if (firstSum == secondSum)
            {
                Console.WriteLine("Yes, sum={0}", firstSum);
            }
            else if (firstSum > secondSum)
            {
                Console.WriteLine("No, diff={0}", firstSum - secondSum);
            }
            else if (secondSum > firstSum)
            {
                Console.WriteLine("No, diff={0}", secondSum - firstSum);
            }
        }
    }
}
